godtools/GeneralExtensions/UIFont.swift

```diff
@@ -77,4 +85,27 @@ extension UIFont {
         return UIFont(name: fontName, size: fontSize)!
     }
     
+    
+    func setBold() -> UIFont {
+        if isBold {
+            return self
+        } else {
+            let fontDescriptorVar: UIFontDescriptor = fontDescriptor.withSymbolicTraits(.traitBold)!
+            return UIFont(descriptor: fontDescriptorVar, size: 0)
+        }
+    }
+    
+    func setItalic()-> UIFont {
+        if(isItalic) {
+            return self
+        } else {
+            let fontDescriptorVar: UIFontDescriptor = fontDescriptor.withSymbolicTraits(.traitItalic)!
```

Comment:
Is there any chance the force unwrap could cause an issue here?

We could always use a guard let to prevent that, and then return something safe if needed.